### ISSUE TITLE
Treeview component guidance considerations and issues

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#3298]
This discussion summarizes considerations and issues identified with the Treeview component which were not regarded as important enough to warrant a bug issue. More details and context can be found in the [Treeview usage ](https://ibm.box.com/s/yqm2l4yf13ix4wzml69us4uq45qenkec)and [Style](https://ibm.box.com/s/fwro22q4l4c0ewdr5cfbcfg7m75t2obj) PDFs.

1. in what situations are treeviews NOT alphabetical at each hierarchy? The Products folder in the screenshot in the Overview section of the Usage is jumbled alphabetically. If alphabetical is the default, that should be included in the Usage guidance. If there are cases where it is not, those should be explained.
2. 

-
